examples/with-turso/tailwind.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
Worth starting with Tailwind v4 alpha or too bleeding edge?

Reply:
If other examples are using it then sure, but if they aren't, maybe wait to update them all. wdyt @leerob